src/gt.ts

```diff
@@ -0,0 +1,37 @@
+import curry from "./curry";
+import Curry from "./types/Curry";
+
+/**
+ * Make sure that it is greater than the reference value.
+ * It does not provide a generic type, so we are going to provide curry by default.
```

Comment:
```suggestion
`Returns true if the first argument is greater than the second; false otherwise.`
```

`curry` is confirmed in the signature, so there is no need to write it separately.

Reply:
It is good suggestion, thx :D 

Even I think it's better to avoid duplicates.